translations/scantailor_de.ts

```diff
@@ -480,7 +480,7 @@ zur Auswahl mehrerer Seiten.</translation>
         <location filename="../MainWindow.cpp" line="1369"/>
         <location filename="../MainWindow.cpp" line="1437"/>
         <source>Scan Tailor Projects</source>
-        <translation type="unfinished">Scan-Tailor-Projekte</translation>
+        <translation type="unfinished">Scan Tailor-Projekte</translation>
```

Comment:
I'd prefer the old variant Scan-Tailor-Projekte, like Friedrich-Ebert-Schule and similar German combined nouns.

Reply:
May be correct, but looks a little bit weird.
Anyway, I have left the old variant and the "unfinished" flag.